src/main/java/drivers/DriverFactory.java

```diff
@@ -33,6 +40,7 @@ public static WebDriver getDriver(BrowserType browser) {
                 driver = getEdge();
                 break;
             default:
+                logger.error("Please, choose a correct name of the browser...");
```

Comment:
I don't think it makes sense to log info when an exception is thrown because we are getting all the info from the exception message.

Reply:
Logger is removed